### DISCUSSION TITLE
Buildcraft pipes

### DISCUSSION THREAD

**Discussion by Intrflex** [RS485/LogisticsPipes#1671] [General]
am i correct in thinking these will still connect with buildcraft pipes? because im unable to get them to connect?

**Comment by Bernasss12** [comment]
I would say you are incorrect since I am able to get them to connect them

![image](https://user-images.githubusercontent.com/6233500/190374933-ddb241b5-221c-4719-8a06-36eaff8963ce.png)

![image](https://user-images.githubusercontent.com/6233500/190375740-37b13273-a6f5-4e5f-88e8-84f57ddb1cf2.png)

**Reply by Intrflex** [reply]
i found the issue, you have to have full buildcraft and not just core + transport

**Reply by bziemons** [reply]
you need at least core + transport + silicon
